mkdocs.yml

```diff
@@ -116,10 +116,13 @@ nav:
                 - Train ML Model:                        tutorials/ml/train-ml-model.md
                 - Predict New Properties:                tutorials/ml/predict-ml-properties.md
             - Python ML:
-                - Train ML Model:                        tutorials/python-ml/train-ml-model.md
+                - Train a ML Regression Model:           tutorials/python-ml/train-regression-model.md
```

Comment:
NIT: is it "a" or "an", let's just could omit it I guess

Reply:
I was mentally pronouncing it as "A machine learning..." but "an" could work here too, since a lot of people pronounce the acronym letter-by-letter.

I'll just drop them here though, to avoid the issue entirely